saleor/plugins/avatax/plugin.py

```diff
@@ -431,13 +428,19 @@ def _calculate_checkout_line_total_price(
             if line.get("itemCode") != item_code:
                 continue
 
+            # The lineAmount does not include the discountAmount, but tax is calculated
+            # for discounted net price, that take into account provided discount.
             tax = Decimal(line.get("tax", 0.0))
+            discount_amount = Decimal(line.get("discountAmount", 0.0))
             net = Decimal(line["lineAmount"])
 
             if currency == "JPY" and tax_included():
-                line_gross = base_value.price_with_sale.gross
+                line_gross = Money(
```

Comment:
Please, make sure that the scenario with JPY currency and discount will also be tested during the QA. We should compare the results values from Avatax and Saleor.